src/viam/sdk/services/motion/server.cpp

```diff
@@ -34,11 +34,8 @@ ::grpc::Status MotionServer::Move(::grpc::ServerContext* context,
                 Motion::constraints::from_proto(request->constraints()));
         }
 
-        const bool success = motion->move(std::move(destination),
-                                          std::move(name),
-                                          std::move(ws),
-                                          std::move(constraints),
-                                          helper.getExtra());
+        const bool success = motion->move(
```

Comment:
Only because `destination` and `name` were declared `const` above.

Reply:
As it turns out, we get stuck in a bit of a jam with `destination` and `name`. Making them un-const causes `misc-const-correctness` to complain, but leaving them const makes `performance-move-const-arg` complain! Thankfully they're only used in this call so we can just avoid assigning an lvalue.